pkg/spec/populators.go

```diff
@@ -200,7 +200,7 @@ func populateVolumes(containers []api_v1.Container, volumeClaims []VolumeClaim,
 				// while deployment since that would not be a complete configuration
 				return nil, fmt.Errorf("neither root level Persistent Volume"+
 					" nor Volume in pod spec defined for %q, "+
-					"in app.containers[%d].volumeMounts[%d]", vm.Name, cn, vn)
+					"in deployment.containers[%d].volumeMounts[%d]", vm.Name, cn, vn)
```

Comment:
This will still be `app`, because of the following reasons:

- this is not tied to deployment in anyway
- user will still be considering it as app
- the error is user facing

Reply:
done